### ISSUE TITLE
Better Downloading Experience

### ISSUE THREAD

**Issue by devhindo** [moovweb/gvm#499]
A cool downloading bar would be great as it looks stuck (I don't know if it's stuck or not) in my caser here.
![image](https://github.com/user-attachments/assets/b4b4ba35-35b4-42a8-8ea3-b0f975a8f555)

[this](https://github.com/schollz/progressbar) might help

**Comment by devhindo** [comment]
it is actually stuck

**Comment by holoyan** [comment]
for me it took 2-3 min to download, and for a moment I thought it was stuck

**Comment by coldsmirk** [comment]
This project hasn't been maintained for two years, so it probably won't add new features.

**Comment by devhindo** [comment]
@ilxqx sadly this is the nature of most of Golang open-source projects

**Comment by coldsmirk** [comment]
> [@ilxqx](https://github.com/ilxqx) sadly this is the nature of most of Golang open-source projects

To be precise, it is the essence of all open-source projects: without sponsorship and income, it is difficult to maintain motivation. Passion alone is often not enough.🤔

**Comment by devhindo** [comment]
@ilxqx yeah but you can see the hype around other open-source project written in Python or JavaScript. Maintained across 10s of years.
